lang/cs/Org.Apache.REEF.Wake/Avro/ProtocolSerializer.cs

```diff
@@ -50,14 +61,12 @@ public sealed class ProtocolSerializer
         /// <summary>
         /// Map from message type (a string with the message class name) to serializer.
         /// </summary>
-        private readonly SortedDictionary<string, Serialize>
-            serializeMap = new SortedDictionary<string, Serialize>();
+        private readonly IDictionary<string, Serialize> serializeMap = new Dictionary<string, Serialize>();
```

Comment:
Okay, makes sense. MS documentation says lookups in dictionary are faster since it is a hash table vs a binary tree for SortedDictionary.